listenbrainz/listenstore/redis_listenstore.py

```diff
@@ -40,3 +40,16 @@ def check_connection(self):
         except redis.exceptions.ConnectionError as e:
             self.log.error("Redis ping didn't work: {}".format(str(e)))
             raise
+
+    def store_latest_listens(self, listens):
+        listens_data = []
+        latest_listens = []
+        for listen in listens:
+            data = self.redis.get('latest_listens:{}'.format(
+                Listen.from_json(timestamp).first))
+            if not data:
+                return None
+        listens_data = data[:10]
+        latest_listens = ujson.loads(listens_data)
+        data.update(latest_listens)
```

Comment:
This won't work, you'll have to do something like redis.setex https://redis.io/commands/setex
To set the value of the latest_listens key

Reply:
for the setex, what value should i write in ```time```parameter?

Reply:
Let's keep it 60 seconds